html-relative-links/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Home Page</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href='index.html'>Home Page</a></li>
+    <li><a href='about-me/about-me.html'>About Me</a></li>
+    <li><a href='projects/projects.html'>Projects</a></li>
+    <li><a href='contact-us/contact-us.html'>Contact Us</a></li>
+  </ul>
+  <p>Hello Homepage!</p>
+  <img src="https://anime-girls-holding-programming-books.netlify.app/static/6b2719eb6b765a3b947addd2eca9f723/dd8b2/Zero_Two_Holding_Nodejs_8_the_Right_Way.png"
```

Comment:
Please use a relative path to the images included in the starter files for this exercise.